lib/shopify-cli/http_request.rb

```diff
@@ -2,11 +2,16 @@
 
 module ShopifyCli
   class HttpRequest
-    def self.call(uri, body, variables, headers)
+    def self.call(uri, body, headers, method)
```

Comment:
I wonder if it makes sense for call to be a shared private method and to expose `get` and `post` methods, they would make the calls more meaningful IMHO.